chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>introduction khatam??</p><p>book k hisab se bahot chhota introduction lag raha hai mereko to.. itna to  blog k hisab se sahi hai..   </p><p> </p><p>A few points</p><ul><li>who, when, why Python was created</li><li>compare with other languages to show why it's better</li><li>brief discussion about the topics that we'll learn in this book</li><li>prerequisites</li><li>expectation from this book</li><li>how to follow this book(like instead of just reading, you should run the code)</li><li>add links of the community</li><li>etc</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
> who, when, why Python was created
compare with other languages to show why it's better

i don't think people would be interested in this when they start about learning a language. Maybe we can add this in the end chapters as a bonus if everything else goes well.

> prerequisites
expectation from this book
how to follow this book(like instead of just reading, you should run the code)
add links of the community

These alongside installation notes would ideally be inside `chapter 0` which is yet to be written. I will try to keep this in mind which i start with it.

> book k hisab se bahot chhota introduction lag raha hai mereko to.. itna to blog k hisab se sahi hai.. 

I don't think size should be a problem. If there are things/topics that need to be covered in detail, that would be better. Any suggestions on that front?

Reply:
<p>Suggestion list</p><ul><li>A list of real life scenarios, preferably something that could potentially be around the user, where programming might be used.</li><li><br></li></ul><p> </p>

Reply:
<p>you're right, the sooner they start writing code the better. otherwise it'd be same like school textbook. </p>